Library/Extensions/Feature+HelpersTests.swift

```diff
@@ -5,31 +5,6 @@ import Prelude
 import XCTest
 
 final class FeatureHelpersTests: TestCase {
-  // MARK: - Qualtrics
-
-  func testFeatureQualtrics_isTrue() {
-    let config = Config.template
-      |> \.features .~ [Feature.qualtrics.rawValue: true]
-
-    withEnvironment(config: config) {
-      XCTAssertTrue(featureQualtricsIsEnabled())
-    }
-  }
-
-  func testFeatureQualtrics_isFalse() {
-    let config = Config.template
-      |> \.features .~ [Feature.qualtrics.rawValue: false]
-
-    withEnvironment(config: config) {
-      XCTAssertFalse(featureQualtricsIsEnabled())
-    }
-  }
-
-  func testFeatureQualtrics_isFalse_whenNil() {
-    withEnvironment(config: .template) {
-      XCTAssertFalse(featureQualtricsIsEnabled())
-    }
-  }
```

Comment:
Just pinged you about the formatter. Give that a run so it fixes spacing issues.